packages/common/src/search/_internal/hubEventsHelpers/processFilters.ts

```diff
@@ -31,20 +46,43 @@ export function processFilters(filters: IFilter[]): Partial<GetEventsParams> {
   if (tags?.length) {
     processedFilters.tags = tags;
   }
+  const readGroupIds = getOptionalPredicateStringsByKey(
+    filters,
+    "readGroupsIds"
+  );
+  if (readGroupIds?.length) {
+    processedFilters.readGroups = readGroupIds;
+  }
+  const editGroupsIds = getOptionalPredicateStringsByKey(
```

Comment:
Similar point, make the IQuery filter name `editGroupId` and update the `hubSearch` docs to reflect this. 

```ts
const editGroupIds = getOptionalPredicateStringsByKey(filters, 'editGroupId');
if (editGroupIds?.length) {
  processedFilters.editGroups = editGroupIds;
}
```